games-arcade/vvvvvv/vvvvvv-2.0~git.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A gravity-flipping, retro 2D puzzle-platformer"
+DESCRIPTION="Notice requested by the author: VVVVVV is a commercial game! The author has \
+given special permission to make this Haiku version available for free. If you \
+enjoy the game, please consider purchasing a copy at thelettervsixtim.es. \
+
+VVVVVV is a hit 2010 indie video game from Super Hexagon creator Terry Cavanagh. \
+Play as Captain Viridian, whose must rescue their crew after a teleporter mishap \
+caused them to be seperated in Dimension VVVVVV."
+HOMEPAGE="https://thelettervsixtim.es"
+COPYRIGHT="Terry Cavanagh 2010-2020"
+LICENSE="VVVVVV Source Code License v1.0"
+REVISION="1"
+srcGitRev="3697487f47c3c4d5cae01c7a5b7e8c4c2212e3c6"
+SOURCE_URI="https://github.com/TerryCavanagh/VVVVVV/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="02c131ceee90f92c72e45aed36cb431705ba03465d3a6f31c302f2dc5be55c7a"
+SOURCE_DIR="VVVVVV-$srcGitRev"
+
+ARCHITECTURES="x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	vvvvvv = $portVersion
+	app:vvvvvv = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libSDL2_mixer_2.0$secondaryArchSuffix
+    vvvvvv_data
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libSDL2_mixer_2.0$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	make$secondaryArchSuffix
+	gcc$secondaryArchSuffix
+	cmake$secondaryArchSuffix
```

Comment:
Should be cmake$secondaryArchSuffix, gcc$secondaryArchSuffix, make$secondaryArchSuffix

Reply:
Does it make a difference in building the package?

Reply:
no, coding guidelines :)